### DISCUSSION TITLE
Storing voltage instead of or in addtion to wattage

### DISCUSSION THREAD

**Discussion by hexagonfounder** [jertel/vuegraf#148] [General]
First, this is a wonderful tool and I really appreciate all of the hardwork.

I am trying to use this to help diagnose voltage fluctions in my home and having the watts in InfluxDB is great for trends. However, as I'm working with my power company they are insisting that I provide them details on the voltage levels not wattage. Any suggestions on how I could get this data into InfluxDB?

**Comment by jertel** [comment]
Hello, I'm not aware of the voltage data being available via the Emporia Via product. If it is it would be at the device level, not at the circuit level, since the circuits only measure current. I haven't used the Emporia App in years and I don't recall if it shows the line voltage. 

Perhaps look for a device like this, https://www.amazon.com/Backlit-Overload-Protection-Kilowatt-Electrical/dp/B0B7LHSCS9/, but make sure it can graph the voltage over time. I don't know if that one will show the voltage; it's just an example.
